README.md

```diff
@@ -234,15 +234,15 @@ Microsoft has implemented a few checks for the software available on Winget to m
 <br><p align="center"><i>Check out the <a href="https://github.com/marticliment/UniGetUI/wiki">Wiki</a> for more information!</i></p>
 
 ## Command-line parameters:
-`--daemon`: Start UniGetUI without spawning a new window. UniGetUI will run minimized on the system tray. UniGetUI is called with this parameter when launched at startup. **Autostart UniGetUI in the notifications area** must be enabled for this parameter to work.<br>
+`--daemon`: Start UniGetUI without spawning a new window. UniGetUI will run as a minimized on the system tray. UniGetUI is called with this parameter when launched at startup. **Autostart UniGetUI in the notifications area** must be enabled for this parameter to work.<br>
```

Comment:
This is not gramatically correct

Reply:
True, then why did Grammarly correct it?

Reply:
Gramarly does not understand well technical language, do not trust it 100%

Reply:
Yeah sure.